catalog/tests/test_models.py

```diff
@@ -0,0 +1,29 @@
+from django.test import TestCase
+
+from catalog.models import Author
+
+class AuthorModelTest(TestCase):
+    @classmethod
```

Comment:
So sánh 2 class methods: `setUp, setUpTestData`

Reply:
`setUpTestData`:
- Chỉ cần gọi 1 lần duy nhất cho toàn bộ các test method
- Dùng để tạo data dùng chung cho tất cả các test method
- Data tạo ra sẽ duy trì trong suốt quá trình chạy các test

Reply:
`setUp`:
- Được gọi trước mỗi test method
- Sử dụng để thiết lập dữ liệu cho từng test method
- Data chỉ có tồn tại trong phạm vi test method đó, sau khi test method kết thúc sẽ bị xoá đi